mavros_msgs/msg/ADSBVehicle.msg

```diff
@@ -0,0 +1,17 @@
+# The location and information of an ADSB vehicle
+#
+# https://pixhawk.ethz.ch/mavlink/#ADSB_VEHICLE
+
+std_msgs/Header header
+
+uint32 ICAO_address
+geographic_msgs/GeoPoint lla
+std_msgs/String altitude_type # Type from ADSB_ALTITUDE_TYPE enum
+uint16 heading  # cog
+uint16 hor_velocity
+int16 ver_velocity
+char[9] callsign  # callsign, 8+null
+std_msgs/String emitter_type  # Type from ADSB_EMITTER_TYPE enum
+std_msgs/Time tslc
+std_msgs/String data_status  # Flags from ADSB_FLAGS
```

Comment:
It's more useful parsed than otherwise. For debugging and for usage. They are INFO fields...